eng/Version.Details.xml

```diff
@@ -388,9 +388,9 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>4665b3d04e1da3796b965c3c3e3b97f55c449a6e</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.Extensions.Telemetry.Testing" Version="9.0.0-alpha.1.23453.3">
+    <Dependency Name="Microsoft.Extensions.Telemetry.Testing" Version="8.0.0-rc.2.23457.7">
```

Comment:
Temporarily "downgrading" since main updates extensions once a week and the current 9.0 version we're using doesn't have the namespace changes.